### PR TITLE
graph random

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/graphs/_MultiplexWU.gen.m [hunk 1]
```diff
@@ -56,6 +66,49 @@ In a multiplex weighted undirected (WU) graph,
...
+attempts_per_edge = g.get('ATTEMPTSPEREDGE');
+
+if nargin<2
+    attempts_per_edge = 5;
+end
+
+% get A
+A = g.get('A');
+L = g.layernumber();
+random_multi_A = cell(1, size(A, 1));
```
Comment (annacanal) [inline braph2genesis/graphs/_MultiplexWU.gen.m:101]: random_multi_A = cell(1, L);
you could use L, shorter

File: braph2genesis/graphs/_MultiplexWD.gen.m [hunk 2]
```diff
@@ -54,6 +64,48 @@ In a multiplex weighted directed (WD) graph,
...
+attempts_per_edge = g.get('ATTEMPTSPEREDGE');
+
+if nargin<2
+    attempts_per_edge = 5;
+end
+
+% get A
+A = g.get('A');
+L = g.layernumber();
+random_multi_A = cell(1, size(A, 1));
```
Comment (annacanal) [inline braph2genesis/graphs/_MultiplexWD.gen.m:99]: random_multi_A = cell(1, L);

File: braph2genesis/graphs/_MultiplexBU.gen.m [hunk 3]
```diff
@@ -56,6 +61,47 @@ In a multiplex binary undirected (BU) graph,
...
+attempts_per_edge = g.get('ATTEMPTSPEREDGE');
+
+if nargin<2
+    attempts_per_edge = 5;
+end
+
+% get A
+A = g.get('A');
+L = g.layernumber();
+random_multi_A = cell(1, size(A, 1));
```
Comment (annacanal) [inline braph2genesis/graphs/_MultiplexBU.gen.m:95]: random_multi_A = cell(1, L);

File: braph2genesis/graphs/_MultiplexBD.gen.m [hunk 4]
```diff
@@ -55,6 +60,47 @@ In a multiplex binary directed (BD) graph,
...
+attempts_per_edge = g.get('ATTEMPTSPEREDGE');
+
+if nargin<2
+    attempts_per_edge = 5;
+end
+
+% get A
+A = g.get('A');
+L = g.layernumber();
+random_multi_A = cell(1, size(A, 1));
```
Comment (annacanal) [inline braph2genesis/graphs/_MultiplexBD.gen.m:94]: random_multi_A = cell(1, L);
you could use L, shorter

**Review by annacanal — CHANGES_REQUESTED** [review]

**Review by annacanal — APPROVED** [review]

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
Please, indent the body of the functions

**Pull request merged**